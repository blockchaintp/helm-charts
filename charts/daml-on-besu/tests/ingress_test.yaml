suite: besu.test the ingresses
templates:
  - jsonapi-ingress.yaml
release:
  name: test
  namespace: testns
tests:
  - it: ingress links to service properly
    set:
      daml:
        jsonapi:
          ingress:
            enabled: true
            hosts:
              - host: test-host
                paths:
                  - /path
      ingress:
        apiVersion: extensions/v1beta1
    asserts:
      - equal:
          path: spec.rules[0].http.paths[0].backend.serviceName
          value: test-daml-on-besu
      - equal:
          path: spec.rules[0].http.paths[0].backend.servicePort
          value: 8080
      - matchSnapshot:
          path: spec
