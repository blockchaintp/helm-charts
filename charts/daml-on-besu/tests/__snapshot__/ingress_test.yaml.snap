ingress links to service properly:
  1: |
    rules:
    - host: test-host
      http:
        paths:
        - backend:
            serviceName: test-daml-on-besu-rpc
            servicePort: 8080
          path: /path
          pathType: ImplementationSpecific
